cypress/integration/ete/registration/register.spec.ts

```diff
@@ -111,17 +111,18 @@ describe('Registration flow', () => {
         receivedAfter: timeRequestWasMade,
       },
     ).then((email) => {
-      const body = email.html.body;
+      const id = email.id ?? '';
```

Comment:
it would be worth failing the test early if email.id is undefined.
You could do the same with body, rather than rely on the assertions failing against empty string

Reply:
a lot more explicit, I like that idea. Will change and resolve the conflicts :)

Reply:
I've refactored the email data extraction into its own dedicated command, everything is a lot cleaner now! (see cypress/support/commands/getEmailDetails.ts)